docs/glossary.rst

```diff
@@ -7,58 +7,75 @@ Glossary
-       For more information about the permission system read
-       :doc:`api/authentication_api`.
+   sheet
+      Sheets are the features of :term:`resources <resource>`. A
+      proposal may for example have the sheet
+      ``adhocracy_core.sheets.title.ITitle`` that allows it to have a
+      title and the sheet ``adhocracy_core.sheets.comment.ICommentable``
+      that allows it to be commented on. A resource is really not much
+      more than the sum of its sheets.
 
-   userid
-      The unique id for one userique id of one :term:`group`: "group:<name>".
+   backend
+   frontend
+      The backend is the part of the software that stores the data.  The
+      frontend on the other hand is in charge of showing the data to
+      users.  Having a clear separation between these makes development
+      simpler and theoretically allows to have more than one frontend,
+      e.g. a website and a mobile app.
 
-   group
-       A set of users. Can be mapped to permission :term:`role`.
+   core
+   customization
+      Not all projects implemented with adhocracy are the same. That
+      is why it is very easy to customize it for each individual
+      project. The shared functionality is called "core" while the
+      special code is called "customization".
 
-   groupid
-       Unique id of one :term:`group`: "group:<name>".
+   process
+      A process :term:`resource` represents a participation process.
+      There are very different kinds of these. Idea Collections, where
+      users can enter proposals and get feedback, and giving feedback on
+      prepared documents, are probaly the most common ones.
 
+   user
    role
-      A set of permissions that can be mapped to :term:`principal`
-
-   roleid
-       Unique id of one permission :term:`role`: "role:<name>".
-
    local role
-       A :term:`role` mapped to a :term:`principal` within a local
-       context and all his children.
+   permission
+      What a user is permitted to do depends on their role.  The roles a
+      user has often depend on the context. Example: Amelia (user) may
+      be the creator (role) of one proposal (context) and therefore
+      permitted to edit it (permission). Proposals that she hasn't
+      created, she may not edit, but she may comment on them.
 
-   DAG
-       Versions of one resource that build a directed acyclic graph.
+   workflow
+   phase
+      Participation :term:`processes <process>` typically have multiple
+      phases: For example, you may want to first publish an
+      announcement, then have the actual participation for some time,
+      and display the results once that is over. This is possible by
+      using workflows.  Workflows can be used with processes, but also
+      with any other kind of :term:`resource`. An important feature of
+      workflows is that you can change the :term:`permissions
+      <permission>` for each role based on the phase.
```

Comment:
The technical term is `state` . We can use `phase` instead because its more common  to use in our team. 
But then we cannot difference anymore between The visible `phase` in the frontend (phase widget) and the real worfklow state in the backend.